roles/edpm_libvirt/tasks/configure.yml

```diff
@@ -144,15 +144,15 @@
   become: true
   loop:
     - {"src": "{{ edpm_libvirt_tls_cert_src_dir }}/tls.crt", "dest": "/etc/pki/libvirt/servercert.pem"}
-    - {"src": "{{ edpm_libvirt_tls_cert_src_dir }}/tls.key", "dest": "/etc/pki/libvirt/private/serverkey.pem"}
+    - {"src": "{{ edpm_libvirt_tls_cert_src_dir }}/tls.key", "dest": "/etc/pki/libvirt/private/serverkey.pem", "mode": "0600"}
```

Comment:
only root will see the pivate key. Good.

Reply:
Yeah, indeed. These permissions are newer recommendations.

Just as some background context: the old implementation of `virt-pki-validate` was not correctly validating the permissions that it was recommending, so if run as root, it was not reporting these permission fails.

The newer `virt-pki-validate` fixes that, hence the errors before this change.